.travis.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     - python: 3.7
 cache: pip
 install:
-- pip install -r requirements.txt
+- pip install -e .
```

Comment:
why does this need to be `-e` for travis purposes?

Reply:
good question, I don't think it does? @j1c ?

Reply:
It doesn't. Can be with or without.

Reply:
ef24d5b